llvm/test/Transforms/VectorCombine/RISCV/vector-interleave2-splat.ll

```diff
@@ -0,0 +1,21 @@
+; NOTE: Assertions have been autogenerated by utils/update_test_checks.py UTC_ARGS: --version 5
+; RUN: opt -S -mtriple=riscv64 -mattr=+v,+m,+zvfh %s -passes=vector-combine | FileCheck %s
+; RUN: opt -S -mtriple=riscv32 -mattr=+v,+m,+zvfh %s -passes=vector-combine | FileCheck %s
```

Comment:
Nit, do we need +zvfh in here and vector-interleave2-splat-e64.ll?

Reply:
No we don't need it. It's fixed now.